packages/schema-type/src/typescript/deprecated/utils/encodeSchema.ts

```diff
@@ -6,6 +6,7 @@ import { SchemaType } from "../SchemaType";
  * Port of `Schema.sol` from `@latticexyz/store`
  * @param schema The schema to encode SchemaType[]
  * @returns The encoded schema as a 32 byte hex string
+ * @deprecated use `schemaToHex` from `protocol-parser` instead
```

Comment:
added this small note because i keep forgetting how the new method is called and the first thing i try is `encodeSchema`; now vscode gives a hint for which method to use as replacement of `encodeSchema`